### PR TITLE
Add PufferPanel template

### PR DESCRIPTION
**Pull request by dada513**

### CODE REVIEW

**Comment by dada513** [comment]
STOP button in pufferpanel stops minecraft server, KILL stops sleeping server starter

**Comment by vincss** [comment]
Looks good to me, thanks !
@iwa any suggestions ?

File: docs/pufferpanel.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      "type": "writefile"
+    },
+    {
+      "target": "eula.txt",
+      "text": "eula=${eula}",
+      "type": "writefile"
+    },
+    {
+      "files": [
+        "https://github.com/vincss/mcsleepingserverstarter/releases/download/v1.1.0/sleepingServerStarter.run",
```
Comment (vincss) [inline docs/pufferpanel.json:30]: Maybe put : 
https://github.com/vincss/mcsleepingserverstarter/releases/latest/download/sleepingServerStarter.run

Comment (iwa) [reply docs/pufferpanel.json:30]: +1

**Review by iwa — CHANGES_REQUESTED** [review]
hey! thanks for the PR!
everything looks fine to me, except that, as vincss mentioned, it would be better if you download the .run file from the latest release, otherwise we would have to update the config file every time we're releasing a new version

**Comment by dada513** [comment]
@iwa @vincss should be fixed. Didn't know github allowed to download latest

**Review by iwa — APPROVED** [review]

**Pull request merged**